README.md

```diff
@@ -16,6 +16,8 @@ The app evaluates the readability of page content by comparing the text(foregrou
 - CSS
 - JavaScript
 
+Please note: If you are not marked as a contributor for this project, then you will need to follow the first two steps. If you are marked as a contributor, then you can clone the repo git clone https://github.com/DBenMoshe/contrast-ratio-repo.git and then follow the rest of the steps in order.
```

Comment:
Just two small changes

1. Let's make the Please note bold. In markdown you can you can bold words like this:

`**word you want to bold goes inside here**`

2. Since `git clone` is a command, let's wrap it in backticks like this 
`git clone https://github.com/DBenMoshe/contrast-ratio-repo.git`